libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.utils.ts

```diff
@@ -208,7 +254,7 @@ export const transformApplicationToParentalLeaveDTO = (
       // TODO: refactor period.endDate to not include time
       to: period.endDate.split('T')[0],
       ratio: Number(period.ratio),
-      approved: true,
+      approved: false,
```

Comment:
At one point we thought this meant approved by employer, but this is updated by VMST when they have approved it.